index.css

```diff
@@ -0,0 +1,617 @@
+  gap: 14px;
+}
+
+.bg-footer,
+.bg-footer-about {
+  background-color: #fff;
+}
+
+.grid-5 {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  text-align: center;
+}
+
+.copyright {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 9px;
+  font-weight: normal;
+  text-align: left;
+  color: #7a7a7a;
+  padding-top: 5px;
+}
+
+.w-max-1:hover {
+  box-shadow: 5px 10px 8px #888;
+  transform: scale(1.01);
+}
+
+a:hover {
+  color: rgb(190, 190, 190);
+}
+
+a:focus {
+  color: rgb(190, 190, 190);
+}
+
+@media screen and (min-width: 768px) {
+  #top-panel {
+    background-color: #3e3e3e;
+    width: 100%;
+    display: block;
+  }
+
+  .logo {
+    display: block !important;
+  }
+
+  .nav-ul {
+    display: flex !important;
+  }
+
+  .nav-text {
+    font-family: 'cocogoose', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 20px;
+    color: #3e3e3e;
+  }
+
+  .left-margin {
+    margin-left: 3rem !important;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    padding: 0 20px;
+  }
+
+  .program-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  #btn {
+    padding-left: 0.8rem;
+    padding-right: 0.8rem;
+  }
+
+  .grid-2 {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .toggle-1 {
+    display: block !important;
+  }
+
+  .toggle-2 {
+    display: none;
+  }
+
+  .toggle-3 {
+    display: block !important;
+  }
+
+  .text-card-1 {
+    /* font-family: cocogoose, sans-serif;
+    font-size: 12px;
+    font-weight: normal;
+    line-height: 16px; */
+    text-align: center;
+  }
+
+  .text-card-2 {
+    /* font-family: cocogoose, sans-serif;
+    font-size: 9px;
+    font-weight: normal;
+    line-height: 16px; */
+    text-align: center;
+  }
+
+  .p-top {
+    padding-top: 2rem !important;
+    background: none !important;
+    text-decoration: underline;
+    text-transform: uppercase;
+  }
+
+  .w-max-3 {
+    max-width: 600px;
+  }
+
+  .grid-4 {
+    display: grid;
+    grid-template-columns: 50% 50%;
+  }
+
+  .bg-footer {
+    background-color: rgb(235, 235, 235);
+  }
+
+  .bg-footer-about {
+    background-color: black;
+  }
+
+  .grid-5 {
+    display: grid;
+    grid-template-columns: 34% 66%;
+    text-align: center;
+    padding: 32px 0 !important;
+  }
+
+  .copyright {
+    /* font-family: 'cocogoose', sans-serif;
+    font-size: 12px;
+    font-weight: bold;
+    text-align: left;
+    color: #e1e1e1; */
+
+    font-family: 'cocogoose', sans-serif;
+    font-size: 9px;
+    font-weight: normal;
+    text-align: left;
+    color: #7a7a7a;
+    padding-top: 5px;
+  }
+
+  a:focus {
+    color: rgb(190, 190, 190);
+    font-weight: 900;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .grid {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    padding: 0 20px;
+  }
```

Comment:
- [ ] Great job with your design. Please make sure you increase the `font-size` of this section of your `Home page`  to make it as close as possible to the Figma design🙏 

Original Design | Your Home Page
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190470120-003afc73-c801-447e-aac8-35cceb969a34.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190470053-453e2e09-3042-40b6-b551-92b30a14cbde.png)